draft-jhoyla-req-mtls-flag.md

```diff
@@ -59,15 +59,44 @@ messages in the wild. A client sending the `request_client_auth` flag in the
 `ClientHello` allows the server to request authentication during the initial
 handshake only when it receives a hint the client will handle the request
 without terminating the connection, and, if unable to authenticate, by sending
-an empty `Certificate` message, per {{?RFC8446, Section 4.4.2}}. This enables a
-number of use cases, for example allowing bots to authenticate themselves when
-mixed in with general traffic.
-
-This flag is intended to be used by clients sending automated traffic, not
-those that are directly operated by (human) users. Clients that set this flag
-should only do so when a client certificate is directly available to them, and
-must not prompt the user or take actions that would prompt the user for example
-trying to read certificates from off-board devices such as smart cards.
+an empty `Certificate` message, per {{?RFC8446, Section 4.4.2}}.
+
+This mechanism is primarily intended to allow web servers to cryptographically
+authenticate web crawlers and other trusted sources of automated traffic
+without prompting human users to present a certificate.
+
+The traffic generated by these sources represents a significant and ever
+increasing fraction of the overall traffic to a given web server. Many web
+servers handle this traffic differently than they handle traffic from web
+browsers operated by human users. On the one hand, human users are much more
+latency sensitive than web crawlers. On the other hand, it may be prudent to
+treat requests from an unknown client with more scrutiny than requests from a
+well-known web-crawler whose behaviour is relatively predictable.
+
+It is therefore common for web servers to attempt to identify trusted sources
+of traffic so that they can be distinguished between one another and,
+especially, from human users. This dynamic creates an incentive for attackers
+to impersonate trusted sources of automated traffic to the server: to confer
+on themselves whatever accommodations are conferred on trusted sources.
+Unfortunately, many of the features used for self-identification are easily
+spoofed {{?I-D.draft-meunier-web-bot-auth-architecture}}.
+
+Authentication of the TLS client is sufficient to thwart such attacks, and
+while there are other approaches to solving this problem (see
+{{alternative-approaches}}), authentication at the TLS layer has several
+desirable features for this use case. [[TODO: Address issue \#12 and add a
+reference here.]]
+
+The mechanism specified in this document may be useful for other use cases.
+However, it may entail security, privacy, or operational considerations that
+are relevant to these use cases but are not directly addressed here.
+
+In particular, the `request_client_auth` flag is not intended to be used by web
```

Comment:
request_client_auth has not been introduced yet.
this may also be a reference to the specific definition section via `{#id}` and `{{id}}`

```suggestion
In particular, `request_client_auth` flag, the TLS flag defined in this document, is not intended to be used by web
```